gen3/bin/kube-roll-all.sh

```diff
@@ -238,39 +244,41 @@ gen3 kube-wait4-pods || true
 
 if g3k_manifest_lookup .versions.wts 2> /dev/null; then
   # this tries to kubectl exec into fence
-  gen3 kube-setup-wts
+  gen3 kube-setup-wts &
 else
   gen3_log_info "not deploying wts - no manifest entry for .versions.wts"
 fi
 
 if g3k_manifest_lookup .versions.mariner 2> /dev/null; then
-  gen3 kube-setup-mariner
+  gen3 kube-setup-mariner &
 else
   gen3_log_info "not deploying mariner - no manifest entry for .versions.mariner"
 fi
 
 if g3k_manifest_lookup '.versions["ws-storage"]' 2> /dev/null; then
-  gen3 kube-setup-ws-storage
+  gen3 kube-setup-ws-storage &
 else
   gen3_log_info "not deploying ws-storage - no manifest entry for '.versions[\"ws-storage\"]'"
 fi
 
 if g3k_manifest_lookup .versions.portal 2> /dev/null; then
-  gen3 kube-setup-portal
+  gen3 kube-setup-portal &
 else
   gen3_log_info "not deploying portal - no manifest entry for .versions.portal"
 fi
 
 gen3_log_info "enable network policy"
-gen3 kube-setup-networkpolicy "enable" || true
+gen3 kube-setup-networkpolicy "enable" || true &
```

Comment:
I think you may actually want...
```suggestion
{gen3 kube-setup-networkpolicy "enable" || true} &
```
...if I'm understanding this right. Otherwise I think the first command will block.

Reply:
cc'iong the original authors:
@m0nhawk @jawadqur

Reply:
There was no response here.
@williamhaley , do you want to push a commit with your suggestion?